### PR TITLE
Change from awgreene to operator-framework

### PR DESCRIPTION
**Pull request by jmrodri**
Signed-off-by: jesus m. rodriguez <jesusr@redhat.com>

### CODE REVIEW

**Comment by jmrodri** [comment]
This PR can wait until the other PRs that are inflight get in to avoid having to rebase all the others.

File: PROJECT [hunk 1]
```diff
@@ -2,7 +2,7 @@ domain: io.operator-framework
 layout:
 - go.kubebuilder.io/v3
 projectName: scope-operator
-repo: awgreene/scope-operator
+repo: operator-frameworke/scope-operator
```
Comment (everettraven) [inline PROJECT:5]:
```suggestion
repo: operator-framework/scope-operator
```

**Review by everettraven — CHANGES_REQUESTED** [review]
all instances seem to now be: `operator-frameworke/...`

Looks like you dropped the `e` at the end of `awgreene` in your command:
`ag -l awgreene | xargs -n 1 sed -i 's/awgreen/operator-framework/g'` --> `ag -l awgreene | xargs -n 1 sed -i 's/awgreene/operator-framework/g'`

**Pull request closed without merging**